packages/slice-machine/src/features/onboarding/OnboardingGuide/OnboardingGuide.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Card,
+  CardContent,
+  ProgressBar,
+  Text,
+  useMediaQuery,
+} from "@prismicio/editor-ui";
+import clsx from "clsx";
+import { confetti as fireConfetti, ConfettiConfig } from "dom-confetti";
+import { useRef, useState } from "react";
+
+import { OnboardingProgressStepper } from "@/features/onboarding/OnboardingProgressStepper";
+import {
+  OnboardingProvider,
+  useOnboardingContext,
+} from "@/features/onboarding/OnboardingProvider";
+import { useOnboardingExperiment } from "@/features/onboarding/useOnboardingExperiment";
+
+import styles from "./OnboardingGuide.module.css";
+
+const confettiConfig: ConfettiConfig = {
+  colors: ["#8E44EC", "#E8C7FF", "#59B5F8", "#C3EEFE"],
+  elementCount: 300,
+  width: "8px",
+  height: "8px",
+  stagger: 0.2,
+  startVelocity: 35,
+  spread: 90,
+  duration: 3000,
+};
+
+const OnboardingGuideContent = () => {
+  const { steps, completedStepCount, isComplete } = useOnboardingContext({
+    onComplete,
+  });
+  const [isVisible, setVisible] = useState(true);
+  const isMediaQueryVisible = useMediaQuery({ min: "medium" });
+  const confettiCannonRef = useRef<HTMLDivElement>(null);
+
+  function onComplete() {
+    const { current: confettiCannon } = confettiCannonRef;
+    if (confettiCannon) fireConfetti(confettiCannon, confettiConfig);
+    setTimeout(() => setVisible(false), confettiConfig.duration);
+  }
+
+  if (!isVisible || !isMediaQueryVisible) return null;
+
+  return (
+    <div className={clsx(styles.container, isComplete && styles.invisible)}>
```

Comment:
Btw, I think this works by chance. I think we should have the container, the visible and the invisible styles, otherwise, the styles in invisible don't necessarily override the ones set in the container.

Reply:
I think it would always work because the order of the rules is correct, but I agree with that approach. Done!